<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWindow</class>
 <widget class="QDialog" name="ColorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Izbor boja</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="6">
    <widget class="QPushButton" name="pushButtonVW">
     <property name="text">
      <string>Izaberi boju</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label4">
     <property name="text">
      <string>Ramer-Douglas-Peucker</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="pushButtonPolygon">
     <property name="text">
      <string>Izaberi boju</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="pushButtonRDP">
     <property name="text">
      <string>Izaberi boju</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label5">
     <property name="text">
      <string>Visvalingam-Whyatt</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Temena</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label3">
     <property name="text">
      <string>Ceo poligon</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Pozadina</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButtonaBackground">
     <property name="text">
      <string>Izaberi boju</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushButtonVertices">
     <property name="text">
      <string>Izaberi boju</string>
     </property>
    </widget>
   </item>
   <item row="9" column="6" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="8">
    <widget class="QFrame" name="framePolygonNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QFrame" name="frameVerticesOld">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QFrame" name="frameRDPNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QFrame" name="frameBackgroundOld">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QFrame" name="framePolygonOld">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QFrame" name="frameVWOld">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QFrame" name="frameVerticesNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QFrame" name="frameRDPOld">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QFrame" name="frameBackgroundNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="8">
    <widget class="QFrame" name="frameVWNew"/>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Stare boje</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nove boje</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButtonDefault">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonaBackground</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>chooseBackground()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonVertices</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>chooseVertices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPolygon</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>choosePolygon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRDP</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>chooseRDP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonVW</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>chooseVW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDefault</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>chooseDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonApply</sender>
   <signal>clicked()</signal>
   <receiver>ColorWindow</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseBackground()</slot>
  <slot>chooseVertices()</slot>
  <slot>choosePolygon()</slot>
  <slot>chooseRDP()</slot>
  <slot>chooseVW()</slot>
  <slot>chooseDefault()</slot>
  <slot>apply()</slot>
 </slots>
</ui>
